hanabi.py

```diff
@@ -62,7 +79,12 @@ def reset(self, seed=None, starting_player=0):
         self.turn_index = 0
         self.turns_left: Optional[int] = None # None until the deck is exhausted, then counts down to 0
         self.strikes = 0
-        self.clues = 8
+        self.clues = MAX_CLUES
+        self.information: List[List[CardInformation]] = [[CardInformation() for _ in range(self.cards_per_player)] for _ in range(self.num_players)]
+
+    @property
+    def cards_per_player(self):
+        return 5 if self.num_players < 3 else 4
```

Comment:
Rename to initial_cards_per_player, because towards the end of a game, a player will have fewer than this many cards